packages/liveblocks-client/src/types/ServerMsg.ts

```diff
@@ -114,6 +114,11 @@ export type UserJoinServerMsg<TUserMeta extends BaseUserMeta> = {
    * endpoint.
    */
   info: TUserMeta["info"];
+
+  /**
+   * Permissions that the user has in the Room.
+   */
+  scopes: string[];
```

Comment:
Another philosophical question: should we expose the exact scopes to other clients? Or should the server translate this to “readonly” before sending it over the wire? I worry that exposing this could be a potential attack vector.

Reply:
We always verify from the token and the token cannot be forged so it's not a security risk to have a them public

Also, it'd be easy to infer anyone's scopes even if they were obfuscated since `isReadonly` matches `room:read` + `room:presence:write` in the documentation

Reply:
It's already public in the REST API, we are using scopes for managing a room permissions.
I think it's ok if our REST API + WS protocol is more low level/granular and that we hide this complexity in the client package. 
When we will release a SDK for the REST API we will do the same, hide the complexity.

Reply:
So @ofoucherot, are you suggesting to switch to `isReadonly: boolean` here, instead of `scopes: string[]`?

Reply:
No, I think it's ok to keep scopes for the Websocket messages. There is no risk exposing that since we are already exposing scopes in the REST API.

I think it's ok if we only simplify/add abstraction in the client.

Reply:
👍